### PR TITLE
Add header and avatar image

### PR DESCRIPTION
**Pull request by AnastasiiaSorina**
[Notion](https://www.notion.so/Hero-add-Header-img-81290f99266843d58951bafc62a69eec).

### CODE REVIEW

File: src/_styles/_hero.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+img {
```
Comment (TaniaZaslon) [inline src/_styles/_hero.scss]: Let's namespace this `img` with class, eg. "header-avatar" to avoid overwriting styles for all `img` on page.

**Review by TaniaZaslon — CHANGES_REQUESTED** [review]

File: src/_styles/_hero.scss (hunk 1 above)
Comment (AnastasiiaSorina) [reply src/_styles/_hero.scss]: ✅ Done [020e0a4](https://github.com/AnastasiSo/Portfolio-Project/pull/1/commits/020e0a4b93deeb5e88cca7572f7b7b6675a15d28)

**Review by TaniaZaslon — APPROVED** [review]

File: src/components/hero/Hero.tsx [hunk 2]
```diff
@@ -15,7 +15,9 @@ const Hero: FunctionComponent = () => {
           <SkillsGroup title="Soft Skills" skills={softSkills} />
         </Column>
         <Column size={6} center>
-          <span>Foto</span>
+          <span>
+            <img className="header-avatar" src="./../img/foto.png" />
```
Comment (TaniaZaslon) [inline src/components/hero/Hero.tsx:19]: Let's add all static images to `/public/static/media` and path for image will be `/static/media/[past name of your file here]`.

**Review by TaniaZaslon — CHANGES_REQUESTED** [review]

**Comment by AnastasiiaSorina** [comment]
Clouse. Because UX design is not relevant anymore.

**Pull request closed without merging**